engine/src/main/java/org/terasology/rendering/nui/internal/LwjglCanvasRenderer.java

```diff
@@ -113,7 +113,12 @@
 
     public LwjglCanvasRenderer(Context context) {
         // TODO use context to get assets instead of static methods
-        this.textureMat = Assets.getMaterial("engine:UITexture").get();
+        try {
+            this.textureMat = Assets.getMaterial("engine:UITexture").get();
+        } catch (java.util.NoSuchElementException noSuch) {
+            // Some extra attention to how this is reported, because this is often the first asset we try to load.
+            throw new RuntimeException("Failing to find engine textures", noSuch);
+        }
```

Comment:
Using `Optional::get` surrounded by `try-catch` is an anti-pattern to me. Instead, use [`Optional::orElseThrow`](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#orElseThrow-java.util.function.Supplier-):

```suggestion
        this.textureMat = 
            Assets.getMaterial("engine:UITexture")
                .orElseThrow(e -> throw new RuntimeException("Failing to find engine textures", e));
```